common/src/main/scala/io/phdata/models/HiveAllocation.scala

```diff
@@ -16,7 +16,10 @@ case class HiveAllocation(
     id: Option[Long] = None,
     directoryCreated: Option[Instant] = None,
     databaseCreated: Option[Instant] = None
-)
+) {
+
+  def isUsingHDFS: Boolean = if (location.startsWith("hdfs://")) true else false
```

Comment:
You can parse the url with this to get the scheme (protocol) https://docs.oracle.com/javase/tutorial/networking/urls/urlInfo.html